### PR TITLE
Swapping gi and gi-sandbox.

### PR DESCRIPTION
**Pull request by dan-brooking**
## Description
This looks like a lot but is simply swapping gi and gi-sandbox directories. Also updating a unit test outside those dirs that was using a helper file in gi-sandbox (now gi).

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi-sandbox/components/profile/BenefitsForm.jsx [hunk 1]
```diff
@@ -140,7 +130,7 @@ export class BenefitsForm extends React.Component {
           />
           <div>
             {this.props.militaryStatus === 'active duty' &&
-              chapter33Check && (
+              this.props.giBillChapter === '33' && (
                 <div className="military-status-info warning form-group">
                   <i className="fa fa-warning" />
```
Comment (timwright12) [inline src/applications/gi-sandbox/components/profile/BenefitsForm.jsx:135]:
```suggestion
                  <i className="fa fa-warning" aria-hidden="true" />
```

Comment (dan-brooking) [reply src/applications/gi-sandbox/components/profile/BenefitsForm.jsx:135]: While I do agree with this, I don't want to make any actual changes in this PR other than swapping the directories.

File: src/applications/gi-sandbox/containers/GiBillApp.jsx [hunk 2]
```diff
@@ -1,111 +1,89 @@
...
+              a comprehensive statement or analysis of applicable statutes,
+              regulations, and case law governing this topic. Rather, it’s a
+              plain-language summary. If you are seeking claims assistance, your
+              local VA regional office, a VA-recognized Veterans Service
+              Organization, or a VA-accredited attorney or agent can help.{' '}
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://www.va.gov/ogc/apps/accreditation/index.asp"
+              >
```
Comment (timwright12) [inline src/applications/gi-sandbox/containers/GiBillApp.jsx:80]:
```suggestion
              aria-label="Search Accredited Attorneys, Claims Agents, or Veterans Service
                Organizations (VSO) Representatives (Opens in new window)">
```

File: src/applications/gi-sandbox/components/profile/BenefitsForm.jsx (hunk 1 above)
Comment (timwright12) [reply src/applications/gi-sandbox/components/profile/BenefitsForm.jsx:135]: Is this going into production?

**Review by timwright12 — APPROVED** [review]

**Pull request merged**